source/diff/diff.cpp

```diff
@@ -748,6 +761,67 @@ void Differ::MatchDebugAndAnnotationInstructions(
   }
 }
 
+std::string Differ::GroupIdsHelperGetName(const IdInstructions& id_to,
```

Comment:
I'm not sure why you need this function.  Why can't you just call `GetSanitizedName` in its place?

Reply:
So I added this (and similar in the next change) to refactor some of the duplicate lambads. However, eventually it turned out that I can't pass it in instead of the lambda (duh), so I had to wrap it in a lambda.

I went ahead and made the function accept the function pointer instead of a lambda so I can use these helpers directly.

Reply:
To answer the question more directly, code reuse. If you look at the next commit, the usage pattern here may be clearer.

Reply:
I checked the next commit.  If I replace all uses of `GroupIdsHelperGetName` with `GetSanitizedName`, the code still compiles.  I see the value in `GroupIdsHelperGetTypeId` because it is more that just calling one other function, it gets the type id as well.  However, This function has the same parameters in the same order, the same return value, and are in the same class.

Reply:
Ack, I had put all the lambads used in that function there as a refactor, but you're right this one case is just a simple forward.